File: rb/utility/make_context.rb
# GregorianLunarCalendar SDK utility: make_context
require_relative '../core/context'
module GregorianLunarCalendarUtilities
  MakeContext = ->(ctxmap, basectx) {
    GregorianLunarCalendarContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// GregorianLunarCalendar SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class GregorianLunarCalendarMakeContext
{
    public static function call(array $ctxmap, ?GregorianLunarCalendarContext $basectx): GregorianLunarCalendarContext
    {
        return new GregorianLunarCalendarContext($ctxmap, $basectx);
    }
}
